### ISSUE TITLE
CocoaPods

### ISSUE THREAD

**Issue by michaelpace** [JV17/YoutubePlayer#6]
Would it be possible to host the project on CocoaPods for easier installation and updating?

**Comment by JV17** [comment]
Yes, I am planning to do a release soon with CocoaPods support! :)
